README.md

```diff
@@ -1 +1,16 @@
-# sal-backend
\ No newline at end of file
+# sal-backend
+
+Mac OS 
+in sal/sal-backend
+mkdir src
+cd src
+ln -s /Users/preeti.dave/sal/sal-backend/aws/ aws
+GOOS=LINUX go build -o github.com/build/user github.com/sal-group/sal-backend/user/user.go
```

Comment:
these above steps don't seem right.. as long sal-backend is under GOPATH and putting code in proper folder hierarchy .. the above steps are not needed... 

```
cd ~/HOME/go/src/github.com/sal-org/sal-backend

GOOS=linux go build -o build/user_photos aws/lambdas/user_photos/user_photos.go

```

Reply:
those steps did not work on mac , i will remove these if it gets confusing